### ISSUE TITLE
[Bug] different api results between instances

### ISSUE THREAD

**Issue by Factral** [iv-org/invidious#2416]
When i am making a standard request to `/v1/search` in different instances, different results are returned, for example:

first option [https://invidious.snopyta.org/api/v1/search?q=Mi+Historia+Entre+Tus+Dedos&sort_by=relevance](https://invidious.snopyta.org/api/v1/search?q=Mi+Historia+Entre+Tus+Dedos&sort_by=relevance) <details>
  <summary>output</summary>

```JSON
[
  {
    "type": "video",
    "title": "Mi Historia Entre Tus Dedos",
    "videoId": "MLGmKzv3taY",
    "author": "Gianluca Grignani",
    "authorId": "UColz1FbIlymB0lfzhESxtow",
    "authorUrl": "/channel/UColz1FbIlymB0lfzhESxtow",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://invidious.snopyta.org/vi/MLGmKzv3taY/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 141221624,
    "published": 1632100294,
    "publishedText": "hace 0 segundos",
    "lengthSeconds": 310,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
  {
    "type": "video",
    "title": "Gianluca Grignani   Mi Historia Entre Tus Dedos 1995) HD",
    "videoId": "gpiKehLush4",
    "author": "Gloria t",
    "authorId": "UCM6iyv_nya15cU8gqf6JUOA",
    "authorUrl": "/channel/UCM6iyv_nya15cU8gqf6JUOA",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://invidious.snopyta.org/vi/gpiKehLush4/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 12007421,
    "published": 1411175494,
    "publishedText": "hace 7 años",
    "lengthSeconds": 307,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
  {
    "type": "video",
    "title": "Mi Historia Entre Tus Dedos",
    "videoId": "VUFihrKERiA",
    "author": "Gianluca Grignani",
    "authorId": "UColz1FbIlymB0lfzhESxtow",
    "authorUrl": "/channel/UColz1FbIlymB0lfzhESxtow",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://invidious.snopyta.org/vi/VUFihrKERiA/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 51159951,
    "published": 1632100294,
    "publishedText": "hace 0 segundos",
    "lengthSeconds": 310,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
  {
    "type": "video",
    "title": "Mi Historia Entre Tus Dedos",
    "videoId": "YwpMvs36CPs",
    "author": "Mickey Taveras",
    "authorId": "UCiyBZly__ZuXgAfgqgiD2vg",
    "authorUrl": "/channel/UCiyBZly__ZuXgAfgqgiD2vg",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://invidious.snopyta.org/vi/YwpMvs36CPs/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 1905109,
    "published": 1632100294,
    "publishedText": "hace 0 segundos",
    "lengthSeconds": 281,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
  {
    "type": "video",
    "title": "Mi Historia Entre Tus Dedos",
    "videoId": "HSgppL9wCmM",
    "author": "Los 4",
    "authorId": "UCJuu7aW1RHfIN539NhrYqig",
    "authorUrl": "/channel/UCJuu7aW1RHfIN539NhrYqig",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://invidious.snopyta.org/vi/HSgppL9wCmM/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 6312924,
    "published": 1632100294,
    "publishedText": "hace 0 segundos",
    "lengthSeconds": 308,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
  {
    "type": "video",
    "title": "Gianluca Grignani - Mi Historia Entre Tus Dedos (LETRA)",
    "videoId": "gfGgDGL3Dn0",
    "author": "TED SAD",
    "authorId": "UChyDKITs_3M1oOyouJoKHZA",
    "authorUrl": "/channel/UChyDKITs_3M1oOyouJoKHZA",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://invidious.snopyta.org/vi/gfGgDGL3Dn0/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 403308,
    "published": 1568941894,
    "publishedText": "hace 2 años",
    "lengthSeconds": 297,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
```
</details>

second option [https://vid.puffyan.us/api/v1/search?q=Mi+Historia+Entre+Tus+Dedos&sort_by=relevance](https://vid.puffyan.us/api/v1/search?q=Mi+Historia+Entre+Tus+Dedos&sort_by=relevance) 
<details>
  <summary>output</summary>

```JSON
[
  {
    "type": "video",
    "title": "Mi Historia Entre Tus Dedos",
    "videoId": "VUFihrKERiA",
    "author": "Gianluca Grignani",
    "authorId": "UColz1FbIlymB0lfzhESxtow",
    "authorUrl": "/channel/UColz1FbIlymB0lfzhESxtow",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://vid.puffyan.us/vi/VUFihrKERiA/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 51159951,
    "published": 1632100117,
    "publishedText": "hace 0 segundos",
    "lengthSeconds": 310,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
  {
    "type": "video",
    "title": "Gianluca Grignani   Mi Historia Entre Tus Dedos 1995) HD",
    "videoId": "gpiKehLush4",
    "author": "Gloria t",
    "authorId": "UCM6iyv_nya15cU8gqf6JUOA",
    "authorUrl": "/channel/UCM6iyv_nya15cU8gqf6JUOA",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://vid.puffyan.us/vi/gpiKehLush4/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 12007376,
    "published": 1411175317,
    "publishedText": "hace 7 años",
    "lengthSeconds": 307,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
  {
    "type": "video",
    "title": "Mi Historia Entre Tus Dedos",
    "videoId": "F9tndcvkwj8",
    "author": "Eslabon Armado",
    "authorId": "UCE7RM8YvsgoJdXvYsWn2wbw",
    "authorUrl": "/channel/UCE7RM8YvsgoJdXvYsWn2wbw",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://vid.puffyan.us/vi/F9tndcvkwj8/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 28185057,
    "published": 1632100117,
    "publishedText": "hace 0 segundos",
    "lengthSeconds": 399,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
  {
    "type": "video",
    "title": "Mi Historia Entre Tus Dedos - Eslabon Armado - “Corta Venas” - (Visualizer)",
    "videoId": "UM6DF65BSg0",
    "author": "Eslabon Armado",
    "authorId": "UCE7RM8YvsgoJdXvYsWn2wbw",
    "authorUrl": "/channel/UCE7RM8YvsgoJdXvYsWn2wbw",
    "videoThumbnails": [
      {
        "quality": "maxres",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/maxres.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "maxresdefault",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/maxresdefault.jpg",
        "width": 1280,
        "height": 720
      },
      {
        "quality": "sddefault",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/sddefault.jpg",
        "width": 640,
        "height": 480
      },
      {
        "quality": "high",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/hqdefault.jpg",
        "width": 480,
        "height": 360
      },
      {
        "quality": "medium",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/mqdefault.jpg",
        "width": 320,
        "height": 180
      },
      {
        "quality": "default",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/default.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "start",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/1.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "middle",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/2.jpg",
        "width": 120,
        "height": 90
      },
      {
        "quality": "end",
        "url": "https://vid.puffyan.us/vi/UM6DF65BSg0/3.jpg",
        "width": 120,
        "height": 90
      }
    ],
    "description": "",
    "descriptionHtml": "",
    "viewCount": 10216779,
    "published": 1608426517,
    "publishedText": "hace 9 meses",
    "lengthSeconds": 399,
    "liveNow": false,
    "premium": false,
    "isUpcoming": false
  },
```
</details>

in short, the order of the videos varies between instances.
in the first one is: `MLGmKzv3taY` and in the second one is: `VUFihrKERiA` _(video not available)_

 I understood that all the instances showed the same answer for the same query, i think is a bug

**Comment by unixfox** [comment]
I don't see any bugs here, there is just two videos with the same title:
<img width="567" alt="image" src="https://user-images.githubusercontent.com/4016501/133966127-a7e61409-b2ce-40be-8c66-30bf5fe6fbf7.png">

The video ID `VUFihrKERiA` (the first one on the screenshot) is just not available in some countries so that's why depending on the location of the instance the video may not appear in the search.

And for the order we are just forwarding the order given by Youtube, we can't do much about it sorry.

Your issue is somewhat a duplicate of https://github.com/iv-org/invidious/issues/92.
